### PR TITLE
Refactored constants into new file, got rid of unnecessary args on so…

### PR DESCRIPTION
**Pull request by abbiehowell**
…me methods, extracted some code into methods

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -123,37 +114,42 @@ class Main {
...
-      this.titleBox.emit('attach');
-      this.invocationsLineGraph.emit('attach');
-      this.map.emit('attach');
-      this.log.emit('attach');
-    });
-    screen.title = 'sls-dev-tools';
-
     await this.table.rows.on('select', (item) => {
-      this.funcName = item.content.split('   ')[0];
-      this.refetch();
```
Comment (BenEllerby) [inline index.js]: do we not want to call the api as soon as we select, waiting for next automatic call seems like the UI would seem laggy.

File: index.js [hunk 2]
```diff
@@ -123,37 +114,42 @@ class Main {
...
+      [this.funcName] = item.content.split('   ');
     });
 
+    // eslint-disable-next-line no-constant-condition
     while (true) {
```
Comment (BenEllerby) [inline index.js]: is setInterval not cleaner?

File: index.js [hunk 3]
```diff
@@ -176,70 +172,53 @@ class Main {
...
-      timestamp < moment(this.endTime).valueOf();
-      timestamp = moment(timestamp).add(this.period, 'seconds').valueOf()
-    ) {
-      if (this.data.MetricDataResults[1].Timestamps.every(
-        (it) => moment(it) !== moment(timestamp),
-      )) {
-        this.data.MetricDataResults[1].Timestamps.push(new Date(timestamp));
-        this.data.MetricDataResults[1].Values.push(0);
+  padInvocationsAndErrorsWithZeros() {
+    // For the invocations and errors data in this.data.MetricDataResults, we will add '0' for each timestamp that doesn't
```
Comment (BenEllerby) [inline index.js]: use */

*/ notation

File: index.js (hunk 1 above)
Comment (abbiehowell) [reply index.js]: yes done

File: index.js (hunk 2 above)
Comment (abbiehowell) [reply index.js]: yes done

File: index.js (hunk 3 above)
Comment (abbiehowell) [reply index.js]: yes done

**Pull request merged**